containers/Connector/Connector.tsx

```diff
@@ -68,6 +69,16 @@ const useConnector = () => {
 		dispatch(setSigner(signer));
 	}, [signer, dispatch]);
 
+	useEffect(() => {
```

Comment:
Probably best to move this to a hook and initialise it only on market pages, I think futures has something more custom doesn't it for rates?